fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraVersioning.java

```diff
@@ -159,6 +162,9 @@ public RdfNamespacedStream getVersionList() {
             throw new RepositoryVersionRuntimeException("This operation requires that the node be versionable");
         }
 
+        servletResponse.addHeader(LINK, "<" + LDP_NAMESPACE + "Resource>;rel=\"type\"");
```

Comment:
Could you use the `Link` methods to create the header instead of manually:
```
Link.Builder builder = Link.fromUri(LDP_NAMESPACE + "Resource").rel("type");
servletResponse.addHeader(LINK, builder.build().toString());